benchmarks/setup/BestPracticesWindows.md

```diff
@@ -0,0 +1,16 @@
+# Best Practices Windows
```

Comment:
There is a doc sub-folder in the benchmark folder, could we move it to that sub-folder OR maybe merge it with the Benchmark readme, where we describe the installation?